### ISSUE TITLE
CS1061 'View' does not contain a definition for 'InitializeComponent' in WPF UserControl

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn#51415]
```
Error	CS1061	'FooView' does not contain a definition for 'InitializeComponent' and no accessible extension method 'InitializeComponent' accepting a first argument of type 'FooView' could be found (are you missing a using directive or an assembly reference?)
```

```cs
namespace N
{
    using System.Windows.Controls;

    public partial class FooView : UserControl
    {
        public FooView ()
        {
            this.InitializeComponent();
        }
    }
}
```

`InitializeComponent` is generated by the compiler, it will be there.

This error is all to familiar for all WPF devs, one of the false errors we have learned to live with.
The false errors hurt beginners significantly as it is very hard for them to know that their code is correct and the bug is in the tool.

**Comment by CyrusNajmabadi** [comment]
I'm going to close this out as there isn't enough information here to act on this.  If you can provide a more substantive repro (perhaps a link to git-commit with the full project) someone could potentially look at this.

**Comment by jnm2** [comment]
@CyrusNajmabadi Here's how to repro. Open https://github.com/GuOrg/Gu.Reactive/blob/1d7d49a3f1704473a5b1306147a43b3b03471296/Gu.Reactive.sln. After about a minute, these errors eventually appear:

![image](https://user-images.githubusercontent.com/8040367/109428411-0f786a00-79c5-11eb-9a99-7439d31f6b33.png)

**Comment by jnm2** [comment]
@CyrusNajmabadi Did I provide the information you're asking for, or is there something else I can do to clarify?

**Comment by CyrusNajmabadi** [comment]
Closing out due to age and lack of reports.
